app/src/main/java/com/door43/translationstudio/tasks/UpdateAppTask.java

```diff
@@ -149,6 +147,8 @@ private void updateBuildNumbers() {
      */
     private void upgradePre115() {
         AppContext.context().deleteDatabase(Library.DATABASE_NAME);
+        TargetTranslation[] targetTranslations = AppContext.getTranslator().getTargetTranslations();
+        TargetTranslationMigrator.migrateFromUSXtoUSFM(targetTranslations);
```

Comment:
If this method is made private and hooked up to run when the target translation manifest is v4 then this and the line above can be removed. We no longer have to manually add a call to migration the target translations for new app versions. The `updateTargetTranslations()` method in this class now handles all of that.